persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/MutationMapper.xml

```diff
@@ -264,6 +267,9 @@
         INNER JOIN genetic_profile ON mutation.GENETIC_PROFILE_ID = genetic_profile.GENETIC_PROFILE_ID
         INNER JOIN sample ON mutation.SAMPLE_ID = sample.INTERNAL_ID
         INNER JOIN gene ON mutation.ENTREZ_GENE_ID = gene.ENTREZ_GENE_ID
+        <if test="excludeFusions != null and excludeFusions == true">
```

Comment:
I'm not sure why this is here - so you can return the mutation type and filter out fusions from the count?

Reply:
Hi @n1zea144 @ao508 @zhx828 ,
I appreciate Angelica's hard work! I went through the PR, but I can not be convinced that we need to introduce this new **_excludeFusions_** flag. 

The mutations are always over counted when there are fusions presented. The reason is that we recored fusions and their partner genes in the same mutation_event table. So, to solve this issue we only need to exclude fusions when querying the number of mutations, and include only fusions when querying the number of fusions. Thus, we will no longer have this issue when fusions are moved out from the mutation_event table into the new structural variant table. 

My PR [https://github.com/cBioPortal/cbioportal/pull/6093](url) can provide a temporary comprehensive solution resolving this issue. From the attached screenshots, you can clearly see the count has been fixed, and plus the fusions are still displayed in oncoprint, in the plots and mutations tab as expected. Also, the new fusion table allows users to query the fusion of interest. 

Please correct me if I misunderstood the requirement, @jjgao.
Thank you all again for spending time to investigate this problem!

Reply:
![mutation-genes-table-fixed](https://user-images.githubusercontent.com/17375786/63654393-ac5ea500-c747-11e9-8e75-29b9bdcb8f7d.png)

Reply:
![mutation-genes-table-without-fixing](https://user-images.githubusercontent.com/17375786/63654426-22fba280-c748-11e9-83cc-466a4df4fc30.png)

Reply:
![fusion-in-oncoprint](https://user-images.githubusercontent.com/17375786/63654500-f09e7500-c748-11e9-87d2-cfbd93ab1564.png)

Reply:
![fusion-in-mutations-tab](https://user-images.githubusercontent.com/17375786/63654510-fdbb6400-c748-11e9-9bff-9bedd7d990d8.png)